backend/src/personnel/personnel.service.ts

```diff
@@ -56,7 +59,8 @@ export class PersonnelService {
     });
 
     try {
-      await this.personnelRepository.update(id, { ...person });
+      // This is a 'save' rather than 'update' to allow for updating many-to-many relations
```

Comment:
Just want to double check that we're not passing in a partial "person" here on save?

Reply:
I don't believe it could be a partial person, since we pull it out of the repository first and replace each property that's changed

Reply:
Right, sorry I could have looked at that :P